### ISSUE TITLE
"Click on the LEARN button above to get started" shown to coaches when viewing student report with no data

### ISSUE THREAD

**Issue by jamalex** [learningequality/ka-lite#4088]
Branch: 0.14.x

Expected: This message should only be shown to the Learner, not to a Coach.

Observed:
![image](https://cloud.githubusercontent.com/assets/618416/8663835/93d52046-2984-11e5-8e29-71fc6c7d853a.png)

Debugging:

It seems the message is shown unconditionally, when the user whose data is being requested has no data to load, even if that's not the currently logged in user.

```javascript
var StudentProgressContainerView = Backbone.View.extend({
    // The containing view
    template: HB.template('student_progress/student-progress-container'),

    initialize: function() {
        this.listenTo(this.collection, 'add', this.add_one);
        
        var self=this;

        this.render();

        this.collection.fetch({
                success: function() {
                     if (self.collection.length == 0) {       //if the student visits the my progress page before attempting any quizes/videos
                              show_message("info", gettext("Click on the LEARN button above to get started on your learning journey."));
                              self.$el.html("");             //this is done to remove the 'Progress Report' header 
                          }
                       }
                   });
    },
```

**Comment by jamalex** [comment]
Putting this on 0.14.x milestone, but thinking of it as a post-release bugfix.

**Comment by rtibbles** [comment]
Fixed in #4091
